pkg/dashboard/adapter/config.go

```diff
@@ -35,7 +35,6 @@ func GenDashboardConfig(srv *server.Server) (*config.Config, error) {
 	dashboardCfg.PublicPathPrefix = cfg.Dashboard.PublicPathPrefix
 	dashboardCfg.EnableTelemetry = cfg.Dashboard.EnableTelemetry
 	dashboardCfg.EnableExperimental = cfg.Dashboard.EnableExperimental
-	dashboardCfg.EnableNonRootLogin = true
```

Comment:
Why?

Reply:
The `EnableNonRootLogin` config has been abandoned in the dashboard side after using the feature flag strategy.